### ISSUE TITLE
Improve schema properties configuration mechanism

### ISSUE THREAD

**Issue by justinccdev** [justinccdev/bsbang-crawler#3]
At the moment, there is a bioschemas.__init__.py.DEFAULT_CONFIG but no easy way to override or replace it for a user (tests have their own mechanism).  Need to make this easier to configure without having to a change a file under source control.  I'm thinking we can keep the python format but have it as conf/bsbang-conf.py or similar.

**Comment by innovationchef** [comment]
In Issue #4 , It is mentioned that - _bsbang-crawl does a very hokey top-level crawl of the JSON-LD captured. This only captures a very small amount of information, mainly because this was for proof of concept_ The current DEFAULT_CONFIG is what you are referring to, right? So do you want a system such that the users can give their own CONFIG for crawl?

**Comment by justinccdev** [comment]
Yes, allowing users to easily put in or override parts of the default config is what this is about.  But currently, no config will allow indexing of the JSON-LD deeper than the very first layer of basic properties (e.g. in a [Bioschemas DataCatalog](http://bioschemas.org/specifications/DataCatalog/specification/) it can grab the description and keywords when properly configured, but will do nothing with the provider since that's an embedded Person/Organization structure (and even deeper, Person itself potentially has lots of complex properties like Person.affiliation).

Handling this relies on some resolution of #4, though that in itself will need some sane way to configure the depth of the search (e.g. that we want DataCatalog.provider but not DataCatalog.provider.affiliation).
